include/psp2/screenshot.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \usergroup{SceScreenShot}
+ * \usergroup{SceScreenshot}
```

Comment:
Revert this commit and change it here too instead: https://github.com/vitasdk/vita-headers/blob/3bafb9a504a51966ff2df5a602da9775373b4409/docs/definitions.dox#L224